extensions/sql-migration/src/dialog/feedbackDialog.ts

```diff
@@ -0,0 +1,184 @@
+								iconWidth: 32,
+								height: 32,
+								width: 32,
+							})
+							.component(),
+						view.modelBuilder
+							.text()
+							.withProperties<azdata.TextComponentProperties>({
+								value: loc.FEEDBACK_DIALOG_HEADING,
+								CSSStyles: {
+									'margin': '0 0 0 10px',
+								},
+							})
+							.component(),
+					])
+					.withLayout({
+						width: '100%',
+						alignContent: 'flex-start',
+						flexFlow: 'row',
+					})
+					.component();
+
+				this._buttonGroup = view.modelBuilder
+					.flexContainer()
+					.withItems([
+						this._createFeedbackButton(view, 0, loc.FEEDBACK_DIALOG_RATING_1),
+						this._createFeedbackButton(view, 1, loc.FEEDBACK_DIALOG_RATING_2),
+						this._createFeedbackButton(view, 2, loc.FEEDBACK_DIALOG_RATING_3),
+						this._createFeedbackButton(view, 3, loc.FEEDBACK_DIALOG_RATING_4),
+						this._createFeedbackButton(view, 4, loc.FEEDBACK_DIALOG_RATING_5),
+					])
+					.withLayout({
+						alignContent: 'flex-start',
+						flexFlow: 'row',
+					})
+					.withProperties<azdata.ComponentProperties>({
+						display: 'inline-flex',
+						ariaLabel: loc.FEEDBACK_DIALOG_HEADING,
+					})
+					.component();
+
+				const feedbackInputBox = view.modelBuilder
+					.inputBox()
+					.withProperties<azdata.InputBoxProperties>({
+						rows: 3,
+						inputType: 'text',
+						multiline: true,
+						placeHolder: loc.FEEDBACK_DIALOG_PLACEHOLDER,
+						CSSStyles: {
+							'white-space': 'normal!important',
+						},
+					})
+					.component();
+
+				feedbackInputBox.onTextChanged(
+					value => this._feedbackText = value);
+
+				const formModel = view.modelBuilder
+					.formContainer()
+					.withFormItems([{
+						components: [
+							{
+								component: headingGroup,
+							},
+							{
+								component: this._buttonGroup,
+							},
+							{
+								component: feedbackInputBox,
+							}
+						],
+						title: ''
+					}])
+					.withLayout({ width: '100%' })
+					.component();
+
+				await view.initializeModel(formModel);
+				await this._buttonGroup.items[0].focus();
+			});
+
+			this._dialog.okButton.label = loc.FEEDBACK_DIALOG_SUBMIT_BUTTON;
+			this._dialog.okButton.onClick(async () => await this._execute());
+
+			this._dialog.cancelButton.label = loc.FEEDBACK_DIALOG_CANCEL_BUTTON;
+			this._dialog.cancelButton.onClick(() => this._cancel());
+
+			azdata.window.openDialog(this._dialog);
+		}
+	}
+
+	private async _execute() {
+		sendSqlMigrationActionEvent(
+			TelemetryViews.FeedbackDialog,
+			TelemetryActions.SendFeedback,
+			{
+				'FeedbackRating': this._feedbackRating?.toString() || '',
+				'FeedbackMessage': this._feedbackText?.substr(0, 500) || '',
+			});
+
+		await vscode.window.showInformationMessage(loc.FEEDBACK_DIALOG_SENT_MESSAGE);
+		this._isOpen = false;
+	}
+
+	private _cancel() {
+		this._isOpen = false;
+	}
+
+	private _createFeedbackButton(view: azdata.ModelView, index: number, ariaLabel: string): azdata.Component {
```

Comment:
from the screenshots, it looks like the image is not centered properly. please adjust the size.